utils/setup.cfg

```diff
@@ -3,3 +3,16 @@
 # 3. If at all possible, it is good practice to do this. If you cannot, you
 # will need to generate wheels for each Python version that you support.
 universal=1
+
+[nosetests]
+tests=../,../roles/openshift_master_facts/test/,test/
```

Comment:
thanks @detiber! I just wanted to understand which direction we're  going. I can help with the Makefile changes, in fact had yet another Makefile in the root to run some new tests we're adding.